MonoGame.Framework.Content.Pipeline/Serialization/Intermediate/ReflectiveSerializer.cs

```diff
@@ -127,6 +134,11 @@ protected internal override void Initialize(IntermediateSerializer serializer)
             }
         }
 
+        public override bool CanDeserializeIntoExistingObject
+        {
+            get { return TargetType.IsClass; }
```

Comment:
You're right, `Array` types don't support it, but they're handled separately by `ArraySerializer`. I think this is correct for `ReflectiveSerializer`.